best-practices/accessibility-best-practices.md

```diff
@@ -1,7 +1,8 @@
 ---
 layout: page
-title: Accessible interfaces
-category: Guides
+title: Accessibility best practices
+category: Best practices
+redirect_from: "http://cfpb.github.io/design-manual/guides/accessible-interfaces.html"
```

Comment:
I think we can just specify the path in `redirect_from` (ie `redirect_from: "/guides/accessible-interfaces.html"`). That makes the redirects work for me locally...